apps/admin-gui/src/assets/i18n/en.json

```
@@ -1256,7 +1256,11 @@
       "FULL_NAME": "Full name",
       "USER_ID": "User ID",
       "USER_TYPE": "User type",
-      "EDIT_NAME": "Edit name"
+      "EDIT_NAME": "Edit name",
+      "MY_VO": "Vo where you are admin",
```

```suggestion
      "MY_VO": "Your organizations",
```
I think this is more suitable for users.